app_flutter/lib/service/dev_cocoon.dart

```diff
@@ -121,6 +121,7 @@ class DevelopmentCocoonService implements CocoonService {
       ..key = (RootKey()..child = (Key()..name = '$commitTimestamp'))
       ..author = _authors[author]
       ..authorAvatarUrl = 'https://avatars2.githubusercontent.com/u/${2148558 + author}?v=4'
+      ..message = 'Fake commit message'
```

Comment:
Since the backend does not send protos, appengine_cocoon need's to be updated to parse message in the json

Reply:
Good catch - done